setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name='django-async-downloads',
-    version='0.5.1',
+    version='0.5.2',
```

Comment:
The `minor` label will cause this to go to `0.6.0`. We can just remove the label, or update this version, I don't mind which.

Reply:
label removed :)